app/src/main/java/com/twilio/voice/quickstart/VoiceActivity.java

```diff
@@ -394,9 +400,20 @@ private void setAudioFocus(boolean setFocus) {
             if (setFocus) {
                 savedAudioMode = audioManager.getMode();
                 // Request audio focus before making any device switch.
-                audioManager.requestAudioFocus(null, AudioManager.STREAM_VOICE_CALL,
-                        AudioManager.AUDIOFOCUS_GAIN_TRANSIENT);
-
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+                    AudioAttributes mPlaybackAttributes = new AudioAttributes.Builder()
+                            .setUsage(AudioAttributes.USAGE_MEDIA)
```

Comment:
How did you select `USAGE_MEDIA`? Based on the definitions I would consider `USAGE_VOICE_COMMUNICATION `

Reply:
Looks like they prefer `USAGE_VOICE_COMMUNICATION` in webrtc https://chromium.googlesource.com/external/webrtc/+/master/webrtc/modules/audio_device/android/java/src/org/webrtc/voiceengine/WebRtcAudioTrack.java#70